### PR TITLE
Validity of unions with a zero-sized field

### PR DESCRIPTION
**Pull request by gnzlbg**

### CODE REVIEW

File: reference/src/validity/unions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Validity of unions
+
+**Disclaimer**: This chapter is a work-in-progress. What's contained here
+represents the consensus from issue [#73]. The statements in here are not (yet)
+"guaranteed" not to change until an RFC ratifies them.
+
+The bit `i` of an union is allowed to have value `v` _iff_ there is a variant of
```
Comment (rkruppe) [inline reference/src/validity/unions.md]: This isn't consensus in #73 and so it shouldn't be written down here.

Comment (gnzlbg) [reply reference/src/validity/unions.md]: Latest commit removes this definition and the rationale.

File: reference/src/validity/unions.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Validity of unions
+
+**Disclaimer**: This chapter is a work-in-progress. What's contained here
+represents the consensus from issue [#73]. The statements in here are not (yet)
+"guaranteed" not to change until an RFC ratifies them.
+
+## Validity of unions with zero-sized fields
+
+A union containing a zero-sized field can contain any value. An example of such
```
Comment (rkruppe) [inline reference/src/validity/unions.md]:
```suggestion
A union containing a zero-sized field can contain any bit pattern. An example of such
```

Comment (rkruppe) [reply reference/src/validity/unions.md]: BTW: still not sure if that's the best way to put it but this change is at least an improvement

**Pull request merged**